components/event_dispatcher/introduction.rst

```diff
@@ -81,17 +81,10 @@ Naming Conventions
 The unique event name can be any string, but optionally follows a few simple
 naming conventions:
 
-* use only lowercase letters, numbers, dots (``.``) and underscores (``_``);
-
-* prefix names with a namespace followed by a dot (e.g. ``kernel.``);
-
-* end names with a verb that indicates what action is being taken (e.g.
-  ``request``).
-
-Here are some examples of good event names:
-
-* ``kernel.response``
-* ``form.pre_set_data``
+* Use only lowercase letters, numbers, dots (``.``) and underscores (``_``);
+* Prefix names with a namespace followed by a dot (e.g. ``store.``);
+* End names with a verb that indicates what action has been taken (e.g.
+  ``order_placed``).
```

Comment:
I like this list of proposals, but in my opinion it mixes mandatory things (characters which can be included in the event name) with recommendations. What about rewording it as follows?

Original:

```
The unique event name can be any string, but optionally follows a few simple
naming conventions:

* Use only lowercase letters, numbers, dots (``.``) and underscores (``_``);
* Prefix names with a namespace followed by a dot (e.g. ``store.``);
* End names with a verb that indicates what action has been taken (e.g.
  ``order_placed``).
```

Proposal:

```
The event name, which must be unique in the application, is a string which
can only contain letters, numbers, dots (``.``) and underscores (``_``).
Optionally, you can follow these naming conventions:

* Use only lowercase letters;
* Prefix names with a namespace followed by a dot (e.g. ``store.*``, ``user.*``);
* End names with a verb that indicates what action has been taken (e.g.
  ``order.placed``, ``user.created``).
```